BitwardenShared/UI/Vault/Vault/ImportLogins/ImportLoginsState.swift

```diff
@@ -0,0 +1,7 @@
+// MARK: - ImportLoginsState
+
+/// An object that defines the current state of a `ImportLoginsView`.
+///
+struct ImportLoginsState: Equatable, Sendable {
+    // MARK: Properties
+}
```

Comment:
🤔 Unless you're foreseeing adding properties to this state struct, what do you think about having a general `VoidState`/`NoState`/`NilState` or something like that that we can reuse on views/processors that have nothing on their state?

Reply:
Yeah that makes sense, but I do plan on adding properties to this next.